src/components/navBar/index.tsx

```diff
@@ -1,17 +1,106 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { USER_SELF } from '../constants/url'
+import navLinks from './navlink'
```

Comment:
Can we also follow CamelCase for the file name as well?
```suggestion
import navLinks from './navLink'
```

Reply:
Fixed